src/test/java/com/sap/oss/phosphor/fosstars/data/github/ReleasesFromGitHubTest.java

```diff
@@ -118,6 +120,9 @@ public void testIfGitHubHasNoReleaseAndTags() throws IOException {
     assertEquals(1, values.size());
     assertTrue(values.has(RELEASED_ARTIFACT_VERSIONS));
     assertTrue(values.of(RELEASED_ARTIFACT_VERSIONS).isPresent());
-    assertTrue(values.of(RELEASED_ARTIFACT_VERSIONS).get().isUnknown());
```

Comment:
I understand that we need the features to be implicitly a known value. But why was this line in test method removed? Why not in other tests like `LgtmDataProviderTest`. 

I am not sure I understand the idea behind this pull request.

Reply:
I'll revert this line and replace `assertTrue` with `assertFalse`.

> I am not sure I understand the idea behind this pull request.

`AbstractValue` is supposed to be used as a base class for known values. Therefore, its `isUnknown()` method should return false and child classes should not be able to change that. It should have been called `AbstractKnownValue`.

Reply:
> I understand that we need the features to be implicitly a known value. But why was this line in test method removed? Why not in other tests like `LgtmDataProviderTest`.

This test case uses a project that has empty list of releases and tags on GitHub. Currently, it is treated as there is no info about releases for this project. But what if a project just has really no releases? In this case the feature should just be empty. It should not be unknown. What do you think?

Reply:
I agree, but 
- But sending an empty list of releases if releases are actually unknown or unfound also does not make sense.

I leave this up to you.

Reply:
> But sending an empty list of releases if releases are actually unknown or unfound also does not make sense. 

The difficult part here is that the provider can't actually distinguish between unknown release info and not found release info. I see you suggest to interpret an empty release info as unknown. I don't think it is good because:

- The current provider doesn't provide a correct value if a project just doesn't have release history
- Unknown values hurt confidentiality of scores. If case no release history actually exists, confidentiality of score would be lower than it actually is. That increases a chance to get `UNCLEAR` label when it is actually clear.